### DISCUSSION TITLE
Njalla DNS working ?

### DISCUSSION THREAD

**Discussion by buave** [simple-login/app#2142] [Self-hosting question]
Hello, 
I'm trying to self host SimpleLogin but I get "rejected by rspamd filter" when I try to get a confirmation mail on my proton mail.
I got this error into postfix log:
`Jul 01 23:02:17 app postfix/smtp[3804]: 39DE142790: to=<notmymail@protonmail.com>, relay=mail.protonmail.ch[185.70.42.128]:25, delay=6.5, delays=0.06/0.04/5.9/0.5, dsn=5.7.1, status=bounced (host mail.protonmail.ch[185.70.42.128] said: 554 5.7.1 rejected by rspamd filter (in reply to end of DATA command))
`

I followed the self host doc. I've tried troubleshoot A1 but no mail is being sent and I get an error.

I wonder if Njalla mon DNS could have something to do with it.

I don't really know where to ask this question, sorry.

**Comment by fm** [comment]
This seems more like it's triggering above the Proton spam threshold. Could be a combination of SPF not set up right, DKIM not set up right, IP reputation, a bunch of other things. Would you be willing to share your SL domain?
